etl/app/server/s3.js

```diff
@@ -149,9 +148,18 @@ function fetchSingleDomain(name, mapping) {
       });
 
       return values;
-    } catch (err) {
-      log.warn(`Sync Domain Values (${mapping.domainName}) failed! ${err}`);
-      return [];
+    } catch (errOuter) {
+      try {
+        return await retryRequest(
```

Comment:
This is to retry the request if there's an error thrown during the request? That's a good idea. In that case, though, we might want to move the `Non-200 response` message out of the `retryRequest` function, maybe just throw an error if the status isn't 200 and handle both cases in the catch block to get rid of that duplication, but I think it's fine for now.

Reply:
The `Non-200 response` message still applies, because the `axios.get` function throws an exception for non 200 responses (atleast for 403 and 404 status codes in my testing). We could add more smarts to the error message (i.e., inspect the caught exception and throw the `Non-200 response` error if it is from bad status code or throw some other error message if the error is something else). 

I thought about maybe getting rid of the `if(res.status !== 200) return await retryRequest(...);` code, but I think I want to leave it in there. It just feels safer and we will already be covered if axios updates their logic to not throw exceptions for non 200s.

Reply:
Oh okay, I didn't realize axios throws exceptions in those cases!